dist/build.mjs

```diff
@@ -22,29 +43,57 @@ const presets = {
   'node-lib': nodeLibPreset
 }
 
-// Host project config file. Must be located in `build` directory of a host project. Default filename is 'config.mjs'
-const configFileName = process.argv[5] ? process.argv[5] : 'config.mjs'
-const configModule = 'file:///' + path.posix.join(process.cwd().replace(/\\/g, '/'), 'build/', configFileName)
+const parseClArgs = () => {
+  let params = clArgDefaults
 
-// Support npm packages
-import path from 'path'
-import chalk from 'chalk'
+  if (process.argv.length > 2) {
+    // Parsing arguments
+    for (let i = 2; i < process.argv.length; i++) {
```

Comment:
it might be cleaner to use something like the npm command-line-args package. I'm also not really sure if it's worth the complexity to be backwards compatible here. I think it makes the code harder to understand. But I'm okay with that being refactored in the future too.

Reply:
That's a very good idea to use `command-line-args`. I also agree that backward compatibility may not be so important since it's a major version that can allow to be incompatible.